vegbilder-client/src/constants/versions.ts

```diff
@@ -1,5 +1,8 @@
 const versionLog = {
-  'v1.3.6': ['Kan sende med vegsystemreferanse i url'],
+  'v1.3.7': [
```

Comment:
Kor blei v1.3.6 av? Eller, eg ser at de har hoppa over versjonar tidlegare også. Litt snodig, men greit. :P

Reply:
Ja, grunnen er egentlig at jeg har surret litt med versjonsnummerering i atlas. Fordi det ikke er mulig å deploye filer med like versjonsnumre valgte jeg å oppdatere til 7, men har innsett at det egentlig er litt dumt. Skal prate med Halvor om versjonering i atlas og hvordan det bør gjøres når man bare har mindre ting man vil deploye for testing og ikke hele features.

Reply:
Etter en prat med Halvor kom jeg fram til at det er like greit å oppdatere versjonsnummeret på hver ny deploy selv om det ikke er snakk om store endringer. Så tror jeg øker det til 8 😄  Men skal høre med Ina når hun har tid om hvordan hun har gjort det.

Reply:
Ok. Ja det er ikkje så farleg for min del altså. Var berre litt overraskande.